utils/docker_gnn/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM openjdk:8
```

Comment:
what is this file for?

Reply:
it is derived from the java docker. Bazel requires this.
There might be better options for this